static/js/scroll_bar.js

```diff
@@ -62,6 +62,8 @@ exports.initialize = function () {
                          "@media (max-width: 775px) { .fixed-app .column-middle { margin-left: " + (7 + sbWidth) + "px !important; } } " +
                          "</style>");
 
+        exports.set_layout_width();
+    } else {
```

Comment:
I am not sure if this actually works, but if we want to call `set_layout_width()` either way, then you could just move it outside the `if` clause, right?

Reply:
I actually wanted feedback on this. This will still run if the width is like -100 or whatever. I'm really not even entirely sure of the purpose of this scrollbar code so I'm not sure if this should actually be a straight call to `set_layout_width` or if it should instead be something like `else if (page_params.fluid_layout_width && sbWidth === 0) {`

Reply:
Nice find!  I see the history here: Before we added the `fluid_layout_width` option, we only needed to call the logic that was extracted to `set_layout_width` if `sbWidth` was nonzero ... if it was zero, the operation would be a noop.

I think it's most coherent for `set_layout_width` to be just called unconditionally outside the `if` statement; can you move it?

(And that explains why this problem never happened for me -- sbWidth > 0 on Linux, so this is a macOS bug; we should explain that clearly in the commit message).

@vinitS101 was an issue opened for this bug that this PR should close?  If not, that's fine, just wanted to check.

Reply:
Issue #12685

Reply:
Updated the commit and commit message